askbot/views/users.py

```diff
@@ -1487,12 +1487,45 @@ def user(request, id, tab_name=None):
         user_logged_in=profile_owner.is_authenticated,
     )
 
+    question_filter = {}
+    if request.user != user:
+        question_filter['is_anonymous'] = False
+
+    if askbot_settings.CONTENT_MODERATION_MODE == 'premoderation':
+        question_filter['approved'] = True
+
+    #
+    # Questions
+    #
+    questions_qs = profile_owner.posts.get_questions(
+                    user=request.user
+                ).filter(
+                    **question_filter
+                ).order_by(
+                    '-points'#, '-thread__last_activity_at' to match sorting with ajax loads
+                ).select_related(
+                    'thread', 'thread__last_activity_by'
+                )
+
+    q_paginator = Paginator(questions_qs, const.USER_POSTS_PAGE_SIZE)
+    questions = q_paginator.page(1).object_list
+    question_count = q_paginator.count
+
+    #
+    # Top answers
+    #
+    a_paginator = profile_owner.get_top_answers_paginator(request.user)
+    top_answers = a_paginator.page(1).object_list
```

Comment:
And, looks like the `top_answers` variable isn't used here, so we can remove L1518